client/src/components/screens/PatientVariant/index.jsx

```diff
@@ -65,99 +65,159 @@ const COLUMN_WIDTHS = {
   TINY: 52,
 };
 
-const REPORT_HEADER_NUCLEOTIDIC_VARIATION = 'Variation nucléotidique';
-const REPORT_HEADER_STATUS_PARENTAL_ORIGIN = 'Statut (origine parentale)';
-const REPORT_HEADER_ALLELIC_FREQUENCY = 'Fréquence allélique';
-const REPORT_HEADER_SILICO_PREDICTION = 'Prédiction in silico';
-const REPORT_HEADER_CLIN_VAR = 'ClinVar';
-
 const getValue = curryRight(get)('');
-const valuePresent = x => (x !== undefined && x.trim());
-const join = sep => a => (isArray(a) ? a.join(sep) : '');
+const valuePresent = x => (x !== undefined && x !== '' && x !== null);
 const isCanonical = t => t.canonical;
-
-// Cell generator for Nucleotidic variation
+const insertCR = lines => lines.flatMap(l => [...l, '\n']);
+const _has = curryRight(has);
+
+/**
+ * Cell generator for Nucleotidic variation column
+ * @param {*} variant
+ * @param {*} gene
+ */
 const nucleotidicVariation = (variant, gene) => {
-  const bdExtString = has(variant, 'variant.bdExt.dbsnp') ? variant.bdExt.dbsnp.join('\n') : '';
-
-  const transcriptsRefSeqIds = c => join(' ')(c.transcripts.filter(isCanonical).map(t => getValue('refSeqId')(t)).filter(valuePresent));
-  const refTranscriptString = join(' ')(variant.consequences.map(transcriptsRefSeqIds).filter(valuePresent));
-
-  const cdnChange = join(', ')(variant.consequences.map(c => getValue('cdnaChange')(c)).filter(valuePresent));
-  const aaChange = join(', ')(variant.consequences.map(c => getValue('aaChange')(c)).filter(valuePresent));
-
-  const str = `${variant.mutationId} ${bdExtString} gène:${getValue('geneSymbol')(gene)} \n ${refTranscriptString}: ${cdnChange} \
-               ${aaChange}`;
-  return str;
+  const lines = [variant.mutationId, '\n'];
+
+  const bdExts = has(variant, 'variant.bdExt.dbsnp') ? variant.bdExt.dbsnp : [];
+  lines.push(...insertCR(bdExts));
+
+  const geneLine = [
+    'Gène: ',
+    {
+      bold: true,
+      value: `${getValue('geneSymbol')(gene)}`,
```

Comment:
You can also write:

```
getValue(gene, 'geneSymbol')
```
Curry allows you to pass as many arguments as you want to. If you know all your arguments on invocation, you don't need to wrap every argument in a separate parenthesis